clickhouse_sqlalchemy/engines/__init__.py

```diff
@@ -1,40 +1,14 @@
 
-from .mergetree import (
+from clickhouse_sqlalchemy.engines.mergetree import (  # noqa: F401
```

Comment:
While I agree with the replacement of `__all__` with `noqa: F401`, I'm not sure what forces absolute imports; especially since they're always one-level (no `from ..<> import <>`)

Reply:
I just prefer explicit paths, but I'll look into this topic in more details.

Reply:
@DaniilAnichin , it is fantastic: the `__all__` module attribute here is important for tests on merge engines to succeed, so it gets back.